src/main/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSpark.java

```diff
@@ -33,16 +33,68 @@
 import java.io.PrintStream;
 import java.util.List;
 
+/**
+ * Spark version of the first pass of the base quality score recalibration.
+ * Generates a recalibration table based on various covariates
+ * (such as read group, reported quality score, machine cycle, and nucleotide context).
```

Comment:
Generates a recalibration table based on various covariates (such as read group, reported quality score, machine cycle, and nucleotide context). 

-->

Generates a recalibration table based on various covariates. The default covariates are read group, reported quality score, machine cycle, and nucleotide context.

Reply:
Fixed.